docker/pg_tests/scripts/configs/wale_compatibility_test_config.json

```diff
@@ -0,0 +1,14 @@
+{"PGSSLMODE": "allow",
+"PGDATABASE": "postgres",
+"PGHOST": "/var/run/postgresql",
+"AWS_ACCESS_KEY_ID": "AKIAIOSFODNN7EXAMPLE",
+"AWS_SECRET_ACCESS_KEY": "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY",
+"AWS_ENDPOINT": "http://s3:9000",
+"AWS_S3_FORCE_PATH_STYLE": "true",
+"WALG_COMPRESSION_METHOD": "brotli",
+"WALG_DELTA_MAX_STEPS": "6",
+"WALG_UPLOAD_CONCURRENCY": "10",
+"WALG_DISK_RATE_LIMIT": "41943040",
+"WALG_NETWORK_RATE_LIMIT": "10485760",
+"WALE_FILE_PREFIX": "file://localhost/tmp",
+"WALE_LOG_DESTINATION": "stderr"}
```

Comment:
Config files have a big common part. I meant that you could place it in a common file. Add needed lines by echo and after that use wrap.sh. But in this PR it's okay.
Almost all tests has
PGSSLMODE": "allow",
"PGDATABASE": "postgres",
"PGHOST": "/var/run/postgresql",
"AWS_ACCESS_KEY_ID": "AKIAIOSFODNN7EXAMPLE",
"AWS_SECRET_ACCESS_KEY": "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY",
"AWS_ENDPOINT": "http://s3:9000",
"AWS_S3_FORCE_PATH_STYLE": "true",
"WALG_COMPRESSION_METHOD": "brotli",
WALG_DELTA_MAX_STEPS": "6",
"WALG_UPLOAD_CONCURRENCY": "10",
"WALG_DISK_RATE_LIMIT": "41943040",
"WALG_NETWORK_RATE_LIMIT": "10485760",